code/src/index.css

```diff
@@ -5,9 +5,157 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0.3),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url("./background-image.jpg");
+  background-repeat: repeat;
+  background-attachment: fixed;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.send-thought {
+  border: 1px solid black;
+  width: 90%;
+  height: 135px;
+  margin: 30px 0;
+  padding: 10px;
+  box-shadow: 5px 5px 1px black;
+  background-color: rgb(240, 236, 236);
+  position: relative;
+}
+
+.send-thought form {
+  display: flex;
+  flex-direction: column;
+}
+
+.send-thought textarea {
+  resize: none;
+  margin: 10px 0;
+}
+
+.send-thought-button {
+  width: 80%;
+  height: 36px;
+  border-radius: 20px;
+  border: none;
+  background-color: #ffadad;
+  font-weight: 600;
+  font-size: 12px;
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+}
+
+.send-thought-button:hover {
+  background-color: rgb(250, 123, 123);
+  cursor: pointer;
+}
+
+.send-heartemoji {
+  font-size: 10px;
+  margin: 0 3px;
+}
+
+.character-count {
+  position: absolute;
+  right: 10px;
+  bottom: 0px;
+  font-size: 12px;
+  font-weight: 650;
+}
+
+.red-length {
+  color: red;
+}
+
+.black-length {
+  color: #000000;
+}
+
+.thoughts-section {
+  width: 95%;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.thought-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
+  height: 135px;
```

Comment:
If you remove the height, it will adjust depending on how long the message written is.